### ISSUE TITLE
Not possible to sort by metric

### ISSUE THREAD

**Issue by Tonkonozhenko** [metabase/metabase#8283]
Hi,

I have a saved metric (named "Content Begin") and I want to sort by it and I'm not able to do it.
If I type my metric manually (sum of column "Content Begin") I'm able to sort by it.

Actual result:
![image](https://user-images.githubusercontent.com/1307646/43958319-a0406fac-9cb3-11e8-8764-f0983fc84eb4.png)

Expected result:
![image](https://user-images.githubusercontent.com/1307646/43958370-d918c144-9cb3-11e8-8315-2ac761210167.png)

**Comment by on99** [comment]
Hi, are there any plans for this to be milestoned? This bug has been around for a while now, and really limits metabase as a data exploration tool.

**Comment by sbelak** [comment]
Yes, this is something you will be able to do with the new query builder shipping in 0.33. For a preview see here: https://metabase.com/blog/a-query-for-your-thoughts/index.html

**Comment by on99** [comment]
@sbelak I don't see how it solves the sorting issue from this blog, but I think I should give 0.33 preview a shot, maybe it solves the issue in another way.

**Comment by sbelak** [comment]
It does. ;) There are also a lot of changes underneath the hood. I've just tried it and it now works as expected.

**Comment by on99** [comment]
That's cool，I will try it out tomorrow. Thank you!

**Comment by on99** [comment]
@sbelak I tried, but it's still not able to sort by metrics. I built 3ae7bffed by docker.

![image](https://user-images.githubusercontent.com/13196393/61546225-b53ec580-aa7b-11e9-9dff-2ba14b8761a9.png)

Can you please take a look?

**Comment by on99** [comment]
@sbelak Well I just deployed 0.33.0-preview2 and found that now it can sort by custom expression, but not common metrics (neither old common metrics migrated from old metabase nor newly created common metrics)

**Comment by dfx413** [comment]
can confirm, just installed 0.33.RC1 and sorting by metric is not possible.

**Comment by mazameli** [comment]
In 0.33.4, this is possible via the table column heading action, and if you do that, the metric will show up in the notebook editor:

![image](https://user-images.githubusercontent.com/2223916/66613045-ea3d4e80-eb78-11e9-8490-0261261f1fbc.png)

However, if you remove that sort step and then try adding it in manually, the metric doesn't show up as a selectable option here:

![image](https://user-images.githubusercontent.com/2223916/66613061-fd501e80-eb78-11e9-9652-cf1eae068b3f.png)

**Comment by npuglisi** [comment]
Sorting with heading column doesn't show to me, I have a custom expression and I can't sort this column at all.

![image](https://user-images.githubusercontent.com/39974597/73013138-7c9af280-3df6-11ea-9363-e85ca6065628.png)

![image](https://user-images.githubusercontent.com/39974597/73013141-802e7980-3df6-11ea-8117-b1848ca2221b.png)

![image](https://user-images.githubusercontent.com/39974597/73013147-845a9700-3df6-11ea-8dac-3e392ef0bcad.png)
